umbraco-cloud/optimize-and-maintain-your-site/optimize-performance/manage-cdn-caching.md

```diff
@@ -2,31 +2,46 @@
 
 After [adding hostnames](../../go-live/manage-hostnames/) to your project, it's possible to configure Content Delivery Network (CDN) Caching. This can be done for all or specific hostnames within your project.
 
-These caching options all relate to the traffic that goes through your hostname from the origin (Umbraco Cloud) to the end-user i.e. the traffic of your website and assets from the webserver to the browser.
+The caching options relate to the traffic that goes through your hostname from the origin (Umbraco Cloud) to the end-user. This is the traffic of your website and assets from the webserver to the browser.
 
 The options that are currently available are:
 
 * Enable Cache (default: On)
-* Cache TTL (default: 120 minutes)
+* Cache TTL: Time to Live (default: 120 minutes)
 * Cache Everything (default: off)
 
 ![Default Options](../../build-and-customize-your-solution/set-up-your-project/project-settings/images/CDN-caching-default.png)
 
-When a new hostname is added to a Project, the default settings will be applied. However, you can change the default settings for your project, so that the new hostnames will get the settings you have chosen. This also means that if you enable caching in the default settings and add a new hostname, that caching will be enabled for that new hostname.
+When a new hostname is added to a Project, the default settings will be applied. However, you can change the default settings for your project, so that the new hostnames will get the settings you have chosen. This means that when enabling caching in the default settings and add a new hostname, that caching is enabled for that new hostname.
```

Comment:
```suggestion
When a new hostname is added to a Project, the default settings will be applied. However, you can change the default settings for your project so that the new hostnames will get the settings you have chosen. This means that when enabling caching in the default settings and adding a new hostname, caching is enabled for that new hostname.
```